Portal/src/Datahub.Portal/i18n/localizations2.json

```diff
@@ -173,7 +173,7 @@
   "Key_Activity_Additional_Information": "Key Activity Additional Information",
   "SFT With Attrition": "SFT With Attrition",
   "Data_Sensitivity": "Data Sensitivity",
-  "Project_Status_Values": "Project Status",
+  "Project_Status_Values": "Initiative Status",
```

Comment:
Needs to go in `localizations2.nrcan.json`

Reply:
split into ssc/nrcan files

Reply:
done